### ISSUE TITLE
Doesn't work with openEmu

### ISSUE THREAD

**Issue by Parseee** [qibinc/JoyMapperSilicon#11]
I can map all the keys in settings of openEmu, but i literally cant control any games

**Comment by MCMi460** [comment]
Have you allowed 'Input Monitoring' for OpenEmu on your Mac?

**Comment by Parseee** [comment]
> Have you allowed 'Input Monitoring' for OpenEmu on your Mac?

yep. I can even map the buttons in settings of open emu, but in gameplay it cant be used. All the buttons work in other apps like terminal and other. I'm trying other emulators

**Comment by MCMi460** [comment]
Correct me if I'm wrong, but I believe OpenEmu does not allow inputs from apps like JoyMapperSilicon or Enjoyable. This may not be the truth, as I haven't tested it myself, but when I was using [pynput](https://github.com/moses-palmer/pynput) for one of my projects a few years back, OpenEmu would not recognize anything I sent via keyboard events, whereas basically everything else got it just fine. ([See this](https://github.com/OpenEmu/OpenEmu/wiki/Troubleshooting:-Input-problems#keyboard-and-gamepad-modifier-apps))
Could apply to JoyMapperSilicon; may not, though. Sorry that I can't assist more with this.
